### ISSUE TITLE
add keyboard shortcut to toggle "Make Unique to Scene"

### ISSUE THREAD

**Issue by lighttroupe** [godotengine/godot-proposals#11456]
### Describe the project you are working on

A project with lots of %unique nodes.

### Describe the problem or limitation you are having in your project

I'd like to avoid grabbing the mouse.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Perhaps **Shift-5** or **Ctrl-Shift-5**

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

N/A

### If this enhancement will not be used often, can it be worked around with a few lines of script?

N/A

### Is there a reason why this should be core and not an add-on in the asset library?

N/A

**Comment by Meorge** [comment]
I think I remember seeing a similar proposal recently, where someone also suggested you could type a % as the first character in a node's name and it would make it a unique node. Both a keyboard shortcut and the node-name approach could be quite handy!

**Comment by sockeye-d** [comment]
You can already set one in the shortcuts menu in the editor settings

**Comment by lighttroupe** [comment]
Good point @sockeye-d thanks!  

It seems that it allows Shift-5 but then it doesn't actually work.  Ctrl-Shift-5 _does_ work.  (Godot 4.2.2 on Linux)

So I guess this issue is about adding one as default.

**Comment by lighttroupe** [comment]
@Meorge Yeah [that one](https://github.com/godotengine/godot-proposals/issues/11386) pairs nicely with this one.

**Comment by Mickeon** [comment]
I am converting this to a discussion as you did not fill the proposal's form properly.
